ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-ref.yaml

```diff
@@ -27,6 +27,8 @@ ref:
       default: "serial"
     - name: EGRESS
       default: "{latency: 15ms}"
+    - name: KUBE_CHECK
```

Comment:
I think KUBE_CHECK is specific to node scenarios, not sure if its needed in all scenarios